Tests/CreateAPITests/GenerateOptionsTests.swift

```diff
@@ -18,22 +18,6 @@ final class GenerateOptionsTests: GenerateBaseTests {
         try compare(package: "petstore-only-schemas")
     }
     
-    func testPestoreChangeFilename() throws {
-        // GIVEN
-        let command = try Generate.parse([
-            pathForSpec(named: "petstore", ext: "yaml"),
-            "--output", temp.url.path,
-            "--package", "petstore-change-filename",
-            "--filename-template", "%0.generated.swift"
-        ])
-        
-        // WHEN
-        try command.run()
-        
-        // THEN
-        try compare(package: "petstore-change-filename")
```

Comment:
Please be sure to remove **Expected/petstore-change-filename** as well as the dependency from in **AllPackages/Package.swift**

Reply:
Is there a way we can automate the population of that file?

Reply:
In theory I think so, but its a little tricky so I don't have an exact answer as to how though